x-pack/plugin/security/cli/src/test/java/org/elasticsearch/xpack/security/cli/CertGenUtilsTests.java

```diff
@@ -100,4 +105,31 @@ public void testIsAnyLocalAddress() throws Exception {
         verifyNoMoreInteractions(address);
     }
 
+    public void testIssuerCertSubjectDN() throws Exception {
+        final ZonedDateTime notBefore = LocalDateTime.now().atZone(ZoneId.of("UTC"));
```

Comment:
This line is failing the forbidden APIs check. You probably want:
```suggestion
        final ZonedDateTime notBefore = ZonedDateTime.now(ZoneOffset.UTC);
```

Reply:
I updated the two dates as follows:

```
        final ZonedDateTime notBefore = ZonedDateTime.now(ZoneOffset.UTC);
        final ZonedDateTime notAfter = ZonedDateTime.parse("2099-12-31T23:23:59.999999+00:00");
```

They passes the forbidden API check:

`
./gradlew :x-pack:plugin:security:cli:forbiddenApisTest
`